sql/core/src/test/resources/sql-tests/results/ansi/interval.sql.out

```diff
@@ -355,15 +355,17 @@ struct<INTERVAL '40 minutes 32 seconds':interval>
 -- !query
 select interval 30 day day
 -- !query schema
-struct<>
+struct<day:interval>
 -- !query output
-org.apache.spark.sql.catalyst.parser.ParseException
+30 days
```

Comment:
This is a positive change. It's pretty weird that `30 days days` works but `30 day day` does not, under ANSI mode.

In general, this PR makes things more lenient.